cve_bin_tool/checkers/gstreamer.py

```diff
@@ -0,0 +1,42 @@
+# !/usr/bin/env python3
+"""
+CVE checker for Gstreamer
+
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-9481/Gstreamer.html
+"""
+from ..util import regex_find
+
+
+def guess_contains(lines):
+    """Tries to determine if a file includes expat
+    """
+    count = 0
+    should_contain = [
+        "http://bugzilla.gnome.org/enter_bug.cgi?product=GStreamer",
+        "LGPL",
```

Comment:
"LGPL" isn't very unique, but I guess in combination with the bugzilla string it's ok as a minor refinement.